client/landing/stepper/declarative-flow/tailored-ecommerce-flow.ts

```diff
@@ -19,7 +19,6 @@ import DesignCarousel from './internals/steps-repository/design-carousel';
 import DomainsStep from './internals/steps-repository/domains';
 import Intro from './internals/steps-repository/intro';
 import ProcessingStep from './internals/steps-repository/processing-step';
-import SetThemeStep from './internals/steps-repository/set-theme-step';
```

Comment:
Can we remove `SetThemeStep` altogether, or is it used elsewhere in other flows?

Reply:
(If so, fine if that happens in a separate PR, just curious!)

Reply:
I think we can! I was preparing a separate PR to make this one simpler, but maybe I can change this one!